k8s/upgrade/src/upgrade_resources/upgrade.rs

```diff
@@ -228,7 +305,7 @@ impl UpgradeResources {
                     let ns = Some(ns.to_string());
                     let role = objects::upgrade_job_cluster_role(ns, self.release_name.clone());
                     let pp = PostParams::default();
-                    match self.cluster_role.create(&pp, &role).await {
+                    match self.get_cluster_role().create(&pp, &role).await {
```

Comment:
btw if these are used within the same module then it's also fine to use field directly rather than fn call.